cmd/ctr/snapshot.go

```diff
@@ -195,22 +196,31 @@ var removeSnapshotCommand = cli.Command{
 
 var prepareSnapshotCommand = cli.Command{
 	Name:      "prepare",
-	Usage:     "prepare gets mount commands for digest",
-	ArgsUsage: "[flags] <digest> <target>",
-	Flags:     []cli.Flag{},
+	Usage:     "prepare a snapshot from a tar file by digest",
+	ArgsUsage: "[flags] digest",
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "target",
```

Comment:
Also when `snapshot-name` is missing, the name just defaults to the target as we have today. I don't want a flag which is required.